scripts/fix-comments.py

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env python3
+
+import sys
+import os
+import subprocess
+import re
+
+if len(sys.argv) != 2 or not sys.argv[1].endswith('.lean'):
+    print("usage: fix-lints.py X.lean")
+    sys.exit(1)
+
+leanfile = sys.argv[1]
+
+is_clean = subprocess.run(
+    ['git', 'status', '--untracked-files=no', '--porcelain'],
+    capture_output=True).stdout.decode().rstrip()
+
+if is_clean != "":
+    print("Certain files tracked by git have uncommitted changes.\n")
+    os.system("git status")
```

Comment:
Why not just print out the stdout you prepared earlier?

Reply:
Well, that's a `--porcelain`, which might look a bit unfamiliar to some people.

Reply:
It's probably still worth passing `--untracked-files=no` this time too, since that's all you asked for.